Gemfile.rails42

```diff
@@ -0,0 +1,12 @@
+source 'https://rubygems.org'
+gemspec
+
+gem 'jruby-openssl', :platforms => :jruby
+
+group :test, :remote_test do
+  # gateway-specific dependencies, keeping these gems out of the gemspec
+  gem 'braintree', '>= 2.0.0'
+  gem 'oauth', '0.4.7'
+end
```

Comment:
@wvanbergen I think it's best to keep them in the `Gemfile`, not add them as dev dependencies. It makes it easier to play around with versions, especially since we still have a Rails dependency and could run into compatibility issues. I think of dev dependencies being for dev tools, whereas this is a special case of a library that's only needed if you want to use a particular gateway.